packages/calcite-components/src/components/menu/menu.scss

```diff
@@ -1,9 +1,18 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden using the component's tag as selector.
+ *
+* @prop --calcite-menu-background-color: Specifies the background color of the component.
+ */
+
 :host {
   @apply flex;
 }
 
 ul {
   @apply inline-flex items-center h-full m-0 p-0;
+  background-color: var(--calcite-menu-background-color, var(--calcite-color-transparent));
```

Comment:
I don't think this token is necessary here. The background color is covered by the menu item